ext/standard/tests/math/round_modes.phpt

```diff
@@ -7,24 +7,32 @@ $modes = [
     "PHP_ROUND_HALF_DOWN" => PHP_ROUND_HALF_DOWN,
     "PHP_ROUND_HALF_EVEN" => PHP_ROUND_HALF_EVEN,
     "PHP_ROUND_HALF_ODD" => PHP_ROUND_HALF_ODD,
+    "PHP_ROUND_CEILING" => PHP_ROUND_CEILING,
+    "PHP_ROUND_FLOOR" => PHP_ROUND_FLOOR,
+    "PHP_ROUND_TOWARD_ZERO" => PHP_ROUND_TOWARD_ZERO,
+    "PHP_ROUND_AWAY_FROM_ZERO" => PHP_ROUND_AWAY_FROM_ZERO,
 ];
 
 $numbers = [
     2.5,
     -2.5,
     3.5,
     -3.5,
-    7,
-    -7,
+    15.51,
+    -15.51,
+    5,
+    -5,
     0.61,
     0.69,
+    0.65,
+    -0.65,
     0,
```

Comment:
```suggestion
    0,
    -0,
```
Perhaps it makes sense to also add negative zero here.

Reply:
For all cases negative zero results in `0`. Also `str_pad` doesn't display negative zero correctly, so it may confuse the results: https://3v4l.org/S8tJr

Diff with negative zero case:
```
--
     		   0.65 => 0
     		  -0.65 => -0
     		      0 => 0
016+ 		      0 => 0
     		 1.9999 => 0
     		-1.9999 => -0
     		 0.0001 => 0
--
     		   0.65 => 1
     		  -0.65 => -1
     		      0 => 0
036+ 		      0 => 0
     		 1.9999 => 2
     		-1.9999 => -2
     		 0.0001 => 0
--
     		   0.65 => 0.7
     		  -0.65 => -0.7
     		      0 => 0
056+ 		      0 => 0
     		 1.9999 => 2
     		-1.9999 => -2
     		 0.0001 => 0
--
     		   0.65 => 0.65
     		  -0.65 => -0.65
     		      0 => 0
076+ 		      0 => 0
     		 1.9999 => 2
     		-1.9999 => -2
     		 0.0001 => 0
--
     		   0.65 => 0.65
     		  -0.65 => -0.65
     		      0 => 0
096+ 		      0 => 0
     		 1.9999 => 1.9999
     		-1.9999 => -1.9999
     		 0.0001 => 0.0001
--
     		   0.65 => 0
     		  -0.65 => -0
     		      0 => 0
117+ 		      0 => 0
     		 1.9999 => 0
     		-1.9999 => -0
     		 0.0001 => 0
--
     		   0.65 => 1
     		  -0.65 => -1
     		      0 => 0
137+ 		      0 => 0
     		 1.9999 => 2
     		-1.9999 => -2
     		 0.0001 => 0
--
     		   0.65 => 0.6
     		  -0.65 => -0.6
     		      0 => 0
157+ 		      0 => 0
     		 1.9999 => 2
     		-1.9999 => -2
     		 0.0001 => 0
--
     		   0.65 => 0.65
     		  -0.65 => -0.65
     		      0 => 0
177+ 		      0 => 0
     		 1.9999 => 2
     		-1.9999 => -2
     		 0.0001 => 0
--
     		   0.65 => 0.65
     		  -0.65 => -0.65
     		      0 => 0
197+ 		      0 => 0
     		 1.9999 => 1.9999
     		-1.9999 => -1.9999
     		 0.0001 => 0.0001
--
     		   0.65 => 0
     		  -0.65 => -0
     		      0 => 0
218+ 		      0 => 0
     		 1.9999 => 0
     		-1.9999 => -0
     		 0.0001 => 0
--
     		   0.65 => 1
     		  -0.65 => -1
     		      0 => 0
238+ 		      0 => 0
     		 1.9999 => 2
     		-1.9999 => -2
     		 0.0001 => 0
--
     		   0.65 => 0.6
     		  -0.65 => -0.6
     		      0 => 0
258+ 		      0 => 0
     		 1.9999 => 2
     		-1.9999 => -2
     		 0.0001 => 0
--
     		   0.65 => 0.65
     		  -0.65 => -0.65
     		      0 => 0
278+ 		      0 => 0
     		 1.9999 => 2
     		-1.9999 => -2
     		 0.0001 => 0
--
     		   0.65 => 0.65
     		  -0.65 => -0.65
     		      0 => 0
298+ 		      0 => 0
     		 1.9999 => 1.9999
     		-1.9999 => -1.9999
     		 0.0001 => 0.0001
--
     		   0.65 => 0
     		  -0.65 => -0
     		      0 => 0
319+ 		      0 => 0
     		 1.9999 => 0
     		-1.9999 => -0
     		 0.0001 => 0
--
     		   0.65 => 1
     		  -0.65 => -1
     		      0 => 0
339+ 		      0 => 0
     		 1.9999 => 2
     		-1.9999 => -2
     		 0.0001 => 0
--
     		   0.65 => 0.7
     		  -0.65 => -0.7
     		      0 => 0
359+ 		      0 => 0
     		 1.9999 => 2
     		-1.9999 => -2
     		 0.0001 => 0
--
     		   0.65 => 0.65
     		  -0.65 => -0.65
     		      0 => 0
379+ 		      0 => 0
     		 1.9999 => 2
     		-1.9999 => -2
     		 0.0001 => 0
--
     		   0.65 => 0.65
     		  -0.65 => -0.65
     		      0 => 0
399+ 		      0 => 0
     		 1.9999 => 1.9999
     		-1.9999 => -1.9999
     		 0.0001 => 0.0001
--
     		   0.65 => 10
     		  -0.65 => -0
     		      0 => 0
420+ 		      0 => 0
     		 1.9999 => 10
     		-1.9999 => -0
     		 0.0001 => 10
--
     		   0.65 => 1
     		  -0.65 => -0
     		      0 => 0
440+ 		      0 => 0
     		 1.9999 => 2
     		-1.9999 => -1
     		 0.0001 => 1
--
     		   0.65 => 0.7
     		  -0.65 => -0.6
     		      0 => 0
460+ 		      0 => 0
     		 1.9999 => 2
     		-1.9999 => -1.9
     		 0.0001 => 0.1
--
     		   0.65 => 0.65
     		  -0.65 => -0.65
     		      0 => 0
480+ 		      0 => 0
     		 1.9999 => 2
     		-1.9999 => -1.99
     		 0.0001 => 0.01
--
     		   0.65 => 0.65
     		  -0.65 => -0.65
     		      0 => 0
500+ 		      0 => 0
     		 1.9999 => 1.9999
     		-1.9999 => -1.9999
     		 0.0001 => 0.0001
--
     		   0.65 => 0
     		  -0.65 => -10
     		      0 => 0
521+ 		      0 => 0
     		 1.9999 => 0
     		-1.9999 => -10
     		 0.0001 => 0
--
     		   0.65 => 0
     		  -0.65 => -1
     		      0 => 0
541+ 		      0 => 0
     		 1.9999 => 1
     		-1.9999 => -2
     		 0.0001 => 0
--
     		   0.65 => 0.6
     		  -0.65 => -0.7
     		      0 => 0
561+ 		      0 => 0
     		 1.9999 => 1.9
     		-1.9999 => -2
     		 0.0001 => 0
--
     		   0.65 => 0.65
     		  -0.65 => -0.65
     		      0 => 0
581+ 		      0 => 0
     		 1.9999 => 1.99
     		-1.9999 => -2
     		 0.0001 => 0
--
     		   0.65 => 0.65
     		  -0.65 => -0.65
     		      0 => 0
601+ 		      0 => 0
     		 1.9999 => 1.9999
     		-1.9999 => -1.9999
     		 0.0001 => 0.0001
--
     		   0.65 => 0
     		  -0.65 => -0
     		      0 => 0
622+ 		      0 => 0
     		 1.9999 => 0
     		-1.9999 => -0
     		 0.0001 => 0
--
     		   0.65 => 0
     		  -0.65 => -0
     		      0 => 0
642+ 		      0 => 0
     		 1.9999 => 1
     		-1.9999 => -1
     		 0.0001 => 0
--
     		   0.65 => 0.6
     		  -0.65 => -0.6
     		      0 => 0
662+ 		      0 => 0
     		 1.9999 => 1.9
     		-1.9999 => -1.9
     		 0.0001 => 0
--
     		   0.65 => 0.65
     		  -0.65 => -0.65
     		      0 => 0
682+ 		      0 => 0
     		 1.9999 => 1.99
     		-1.9999 => -1.99
     		 0.0001 => 0
--
     		   0.65 => 0.65
     		  -0.65 => -0.65
     		      0 => 0
702+ 		      0 => 0
     		 1.9999 => 1.9999
     		-1.9999 => -1.9999
     		 0.0001 => 0.0001
--
     		   0.65 => 10
     		  -0.65 => -10
     		      0 => 0
723+ 		      0 => 0
     		 1.9999 => 10
     		-1.9999 => -10
     		 0.0001 => 10
--
     		   0.65 => 1
     		  -0.65 => -1
     		      0 => 0
743+ 		      0 => 0
     		 1.9999 => 2
     		-1.9999 => -2
     		 0.0001 => 1
--
     		   0.65 => 0.7
     		  -0.65 => -0.7
     		      0 => 0
763+ 		      0 => 0
     		 1.9999 => 2
     		-1.9999 => -2
     		 0.0001 => 0.1
--
     		   0.65 => 0.65
     		  -0.65 => -0.65
     		      0 => 0
783+ 		      0 => 0
     		 1.9999 => 2
     		-1.9999 => -2
     		 0.0001 => 0.01
--
     		   0.69 => 0.69
     		   0.65 => 0.65
     		  -0.65 => -0.65
802+ 		      0 => 0
     		      0 => 0
     		 1.9999 => 1.9999
     		-1.9999 => -1.9999
--
```

Reply:
I excluded tests for zeroes to separate test file.